biome.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
This is the existing [Biome](https://biomejs.dev/) config that @calvinbrewer set up (that now gets checked in CI).